### ISSUE TITLE
Hostname blank for declarative onboard if mgmt_ip set to false

### ISSUE THREAD

**Issue by Pacobart** [F5Networks/terraform-aws-bigip-module#8]
When setting "public_ip" to false for management, the terraform output for declarative onboarding is blank. This causes declarative onboarding to fail due to no hostname beint set.
```
mgmt_subnet_ids             = [{ "subnet_id" = aws_subnet.mgmt.id, "public_ip" = false, "private_ip_primary" = null }]
```

Proposed fix: https://github.com/F5Networks/terraform-aws-bigip-module/pull/7

**Comment by RavinderReddyF5** [comment]
@Pacobart please find CLA here: https://github.com/F5Networks/terraform-provider-bigip/blob/master/F5_Contributor_License_Agreement.pdf

**Comment by RavinderReddyF5** [comment]
@Pacobart fixed through v1.1.3

**Comment by Pacobart** [comment]
@RavinderReddyF5 thank you for fixing this!
